components/Tabs/src/MDCTabBarViewController.m

```diff
@@ -68,6 +75,29 @@ @implementation MDCTabBarViewController {
   MDCTabBarShadowView *_tabBarShadow;
 }
 
+- (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
+  self = [super initWithCoder:aDecoder];
+  if (self) {
+    _viewControllers = [aDecoder decodeObjectOfClass:[NSArray class]
```

Comment:
Should this (and the assignments below) use the custom setters below?

Reply:
Ops! this is correct, done